src/Components/Authentication/FormSwitcher.tsx

```diff
@@ -165,6 +166,7 @@ export class FormSwitcher extends React.Component<FormSwitcherProps, State> {
 
     return (
       <Form
+        contextModule={this.props.options.contextModule}
```

Comment:
I actually think I need to pass the `intent` versus the `contextModule` per the spreadsheet in the ticket.